app/assets/src/components/views/amr_heatmap/AMRHeatmapVis.jsx

```diff
@@ -0,0 +1,262 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Heatmap from "~/components/visualizations/heatmap/Heatmap";
+import { getAMRCounts } from "~/api/amr";
```

Comment:
We have a convention where we group our imports into three groups with a space between them, for readability. 
1) 3rd party libraries, 2) imports starting with `~/` 3) imports starting with `../` or `./`